src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -25,7 +25,7 @@ export class TaskListBuilder {
     )
 
     const now: Moment = MomentFactory.currentDateTime()
-    if (claim.responseDeadline.isAfter(now)) {
+    if (claim.responseDeadline.endOf('day').isAfter(now)) {
```

Comment:
On the second thought in the frontend we say:

`'You’ll have until 4pm on ' + claim.responseDeadline.format('LL') + ' to respond'`

Can you ask business whether we to use end of day or 4pm as deadline?

Reply:
Will do that now.

Reply:
It has to be 4 PM @geek-soft, very good spot. I also need to check if CCJ is working correctly with regard to this.